packages/elements/index.html

```diff
@@ -1,14 +1,19 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Logto elements dev page</title>
   <script type="module" src="./src/index.ts"></script>
 </head>
+
 <body style="background: #111;">
   <logto-theme-provider theme="dark">
-    <logto-profile-card></logto-profile-card>
+    <logto-user-provider user='{"name": "Johnny Silverhand", "avatar": "https://github.com/logto-io.png"}'>
```

Comment:
Is "Johnny Silverhand" used for placeholder?

Reply:
it's removed in #6284

Reply:
OK, feels better that way